src/libraries/System.ServiceProcess.ServiceController/src/CompatibilitySuppressions.xml

```diff
@@ -1,15 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <Suppression>
-    <DiagnosticId>PKV006</DiagnosticId>
-    <Target>.NETStandard,Version=v1.4</Target>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceProcess.ServiceController.Stop(System.Boolean)</Target>
+    <Left>lib/netcoreapp3.1/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Right>
```

Comment:
This looks like a bug in the package.

There is no longer a rid-less `netcoreapp` TFM in the latest package due to 07143d4d6d527251849a1aaa2909bcbf46b59fbe

Package validation is comparing the old package's `netcoreapp3.1` reference asset against the latest packages `netstandard2.0` asset, since that's the applicable one, and noticing the missing API.

Reply:
I don't think this is a bug on the package. This was an intentional removal of the netcoreapp3.1 assets on the package, because we are now dropping assets that will no longer be supported when 7.0 ships.

https://github.com/dotnet/runtime/pull/58011

Reply:
I see, so long as we aren't removing API from a supported TFM by dropping that asset from a not supported TFM it's OK.  It wasn't clear to me if that was the case.  I had assumed this tuple came up because it was testing some supported framework in the old package against a supported framework in the new package (eg: net5.0) and resolved different compile assets with different API.  I guess we really need the suppression around "don't compare dropped frameworks, instead compare next compatible framework"

Reply:
Yup. I'll open an issue to track this. I imagine an item group to suppress tfms. We can discuss that in the issue.

Reply:
I opened: https://github.com/dotnet/sdk/issues/22901